src/js/core/widget/core/Popup.js

```diff
@@ -644,6 +697,8 @@
 				eventUtils.on(self._ui.page, "pagebeforehide", self, false);
 				eventUtils.on(window, "resize", self, false);
 				eventUtils.on(document, "vclick", self, false);
+
```

Comment:
in this place:
`eventUtils.on(self._ui.content, "scroll", self, false);`

Reply:
done.